### ISSUE TITLE
Benchmarks?

### ISSUE THREAD

**Issue by jeremyherbert** [python-ivi/python-vxi11#9]
Hi,

I'm trying to pull waveform data off a R&S RTO 1014 via gigabit ethernet. At the moment, it is receiving data rather slowly (10Mbit), whereas if I connect an agilent scope with USB and usbtmc, I can push close to the limit of 480MBit (I estimate something like 350MBit). Unfortunately this R&S one doesn't have USB, so I am stuck with ethernet.

Are there any benchmarks on what sort of data rate I can expect using python-vxi11? For me right now, 15MB of waveform data takes about 6 seconds on a point-to-point gigabit ethernet connection (no router or switches in between) which is far too slow. I have checked and the interface is definitely running at 1000MBit. 

I'm running Ubuntu 15.04 x64 on a Core i3 box, using the intel gigabit adapter on the motherboard.

**Comment by jeremyherbert** [comment]
Just as an update to this, the command:

`echo ":channel1:data?" | nc.traditional xxx.xxx.xxx.xxx 5025 > /dev/null`

gets up to 100MBit/s

**Comment by alexforencich** [comment]
Right, there was a performance issue reported a while ago related to the use of (bytes) += instead of (bytearray).append.  I will look in to reworking that as soon as I am back in town and can test it out properly.

**Comment by alexforencich** [comment]
Try again with the latest version of python-vxi11 and let me know if you are still seeing the same issue.
